examples/trials/mnist-pbt-tuner-pytorch/mnist.py

```diff
@@ -0,0 +1,186 @@
+import os
+import argparse
+import logging
+import nni
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+
+
```

Comment:
can you merge this minst-example and other examples as one? I cannot tell any difference between them.

Reply:
They are different, pbt requires users to explicitly load and save checkpoint in the trial code.